jose-jwt/jose-jwt.csproj

```diff
@@ -77,7 +77,6 @@ JSON Web Key (RFC 7517):
     <NeutralLanguage>en-US</NeutralLanguage>
     <VersionPrefix>4.1.0</VersionPrefix>
     <Authors>Dmitry Vsekhvalnov</Authors>
-    <TargetFrameworks>net40;net461;net472;netstandard1.4;netstandard2.1</TargetFrameworks>

```

Comment:
not sure why we want to drop targets here?

Reply:
It wasn't on purpose, I'll rollback.